modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -3,10 +3,25 @@
 
 Because the monitoring agent requires a SOCKS5 proxy, you need two proxy types if you use Anypoint Monitoring.
 
+[NOTE]
+Starting Runtime Fabric version 2.6.22, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version less than 2.6.22.
```

Comment:
SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version 2.6.22 onwards.